<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTransformRecorderModule</class>
 <widget class="qSlicerWidget" name="qSlicerTransformRecorderModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>345</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="InputGroupBox">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="ModuleNodeLabel">
           <property name="text">
            <string>Module Node:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="ModuleComboBox">
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLTransformRecorderNode</string>
            </stringlist>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="TransformSelectionsLabel">
           <property name="text">
            <string>Transform Selection(s):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="qMRMLCheckableNodeComboBox" name="TransformCheckableComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="nodeTypes">
            <stringlist>
             <string>vtkMRMLLinearTransformNode</string>
            </stringlist>
           </property>
           <property name="addEnabled">
            <bool>false</bool>
           </property>
           <property name="removeEnabled">
            <bool>false</bool>
           </property>
           <property name="renameEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="ControlsGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="BufferGroupBox">
        <property name="title">
         <string>Buffer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="NumRecordsLabel">
           <property name="text">
            <string>Number of Records:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="TotalTimeLabel">
           <property name="text">
            <string>Total Time (s):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="TotalNeedlePathLabel">
           <property name="text">
            <string>Total Needle Path (mm):</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QLabel" name="InsideNeedlePathLabel">
           <property name="text">
            <string>Inside Needle Path (mm):</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="TotaNeedlelPathResultsLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="InsideNeedlePathResultsLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="TotalTimeResultsLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="NumRecordsResultLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="MessagesGroupBox">
        <property name="title">
         <string>Messages</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLCheckableNodeComboBox</class>
   <extends>qMRMLNodeComboBox</extends>
   <header>qMRMLCheckableNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerTransformRecorderModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ModuleComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerTransformRecorderModule</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>TransformCheckableComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
